clusters/app.ci/supplemental-ci-images/coreos/fcos-buildroot.yaml

```diff
@@ -1,14 +1,41 @@
-# See https://github.com/coreos/fedora-coreos-config/pull/740
-apiVersion: image.openshift.io/v1
-kind: ImageStream
-metadata:
-  name: fcos-buildroot
-  namespace: coreos
-spec:
-  tags:
-  - name: testing-devel
-    from:
-      kind: DockerImage
-      name: quay.io/coreos-assembler/fcos-buildroot:testing-devel
-    importPolicy:
-      scheduled: false # mirrored
+# # See https://github.com/coreos/fedora-coreos-config/pull/740
+
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: image.openshift.io/v1
+    kind: ImageStream
+    metadata:
+      name: fcos-buildroot
+      namespace: coreos
+    spec:
+      tags:
+        - name: testing-devel
+  - kind: BuildConfig
+    apiVersion: v1
+    metadata:
+      namespace: coreos
+      name: fcos-buildroot
+    spec:
+      successfulBuildsHistoryLimit: 1
+      failedBuildsHistoryLimit: 2
+      output:
+        to:
+          kind: ImageStreamTag
+          name: fcos-buildroot:testing-devel
+      source:
+        git:
+          uri: https://github.com/coreos/fedora-coreos-config.git
```

Comment:
This needs a `ref` here for the webhook to work apparently. (See https://github.com/openshift/release/pull/26838.)

Reply:
Good catch, thank yoU!